src/site/stages/build/drupal/graphql/CountEntityTypes.graphql.js

```diff
@@ -24,12 +24,31 @@ const CountEntityTypes = `
   vetCenters: nodeQuery(
     filter: {
       conditions: [
-        # TODO: uncomment when vet_centers officially go live:
-        # {field: "status", value: ["1"]},
+        {field: "status", value: ["1"]},
```

Comment:
We are using Tugboat now for local development, so we can filter out unpublished vet center data.